packages/core/helper-plugin/src/features/CustomFields.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+
+import { MessageDescriptor } from 'react-intl';
+
+/**
+ * @TODO: Custom field types should be defined and exported from the registry:
+ * packages/core/admin/admin/src/core/apis/CustomFields.js
```

Comment:
![Excited Season 4 GIF by The Office](https://media4.giphy.com/media/VMO6qeIbr7JRLnLTGw/200.gif)